content/300-guides/050-database/100-developing-with-prisma-migrate/170-customizing-migrations.mdx

```diff
@@ -159,104 +161,104 @@ To change the direction of a 1-1 relation:
 
    ```prisma
    model User {
-        id        Int      @id @default(autoincrement())
-        name      String
-        posts     Post[]
-        profile   Profile? @relation(fields: [profileId], references: [id])
-        profileId Int
+     id        Int      @id @default(autoincrement())
+     name      String
+     posts     Post[]
+     profile   Profile? @relation(fields: [profileId], references: [id])
+     profileId Int
    }
-
-    model Profile {
-        id        Int @id @default(autoincrement())
-        biography String
-        user      User
-    }
-    ```
+   
+   model Profile {
+     id        Int    @id @default(autoincrement())
+     biography String
+     user      User
+   }
+   ```
 
 1. Run the following command to create a **draft migration** that you can edit before applying to the database:
 
-    <CodeWithResult expanded={true}>
-    <cmd>
+   <CodeWithResult expanded={true}>
+   <cmd>
```

Comment:
Double check in the rendered version that this still works with the changed indentation.